coderd/database/gentest/models_test.go

```diff
@@ -51,10 +51,10 @@ func TestViewSubsetTemplateVersion(t *testing.T) {
 	}
 }
 
-// TestViewSubsetWorkspaceBuild ensures WorkspaceBuildTable is a subset of WorkspaceBuild
+// TestViewSubsetWorkspaceBuild ensures WorkspaceBuild is a subset of WorkspaceBuild
 func TestViewSubsetWorkspaceBuild(t *testing.T) {
 	t.Parallel()
-	table := reflect.TypeOf(database.WorkspaceBuildTable{})
+	table := reflect.TypeOf(database.WorkspaceBuild{})
```

Comment:
This change doesn't look too good. This is to ensure the view is always correct, so revert this change. The test should still pass.

Reply:
Good catch, went overboard on the find and replace